test_conformance/images/kernel_read_write/test_iterations.cpp

```diff
@@ -1171,6 +1171,25 @@ bool validate_float_write_results( float *expected, float *actual, image_descrip
     return pass;
 }
 
+bool validate_half_write_results( cl_ushort *expected, cl_ushort *actual, image_descriptor *imageInfo )
+{
+    bool pass = true;
+    // Compare half floats
+    if (memcmp(expected, actual, 2 * get_format_channel_count(imageInfo->format)) != 0) {
```

Comment:
s/2/sizeof(cl_half) :).

Reply:
This also applies to validate_float_write_results. Should I fix that one as well?

Reply:
If you don't mind. It's a small-ish thing but could prevent some head-scratching.